src/components/activity-header/activity-nav-header.tsx

```diff
@@ -16,17 +15,9 @@ interface IProps {
 
 export class ActivityNavHeader extends React.PureComponent <IProps> {
   render() {
-    const { activityName, activityPages, currentPage, fullWidth, onPageChange, singlePage} = this.props;
```

Comment:
activityName can be taken out of the component props since it is no longer used.